kernel/arch/dreamcast/include/dc/math.h

```diff
@@ -0,0 +1,27 @@
+/* KallistiOS ##version##
+
+   dc/math.h
+   Copyright (C) 2023 Paul Cercueil
+
+*/
+
+#ifndef __DC_MATH_H
+#define __DC_MATH_H
+
+#include <sys/cdefs.h>
+__BEGIN_DECLS
+
+/**
+    \file   dc/math.h
+    \brief  Prototypes for optimized math functions written in ASM
+    \author Paul Cercueil
+*/
+
+/**
+    \brief  Returns the bit-reverse value of the argument (where MSB
+            becomes LSB and vice-versa).
+    \return the bit-reverse value of the argument.
+*/
+unsigned int bit_reverse(unsigned int value);
+
+#endif /* __DC_RBIT_H */
```

Comment:
__DC_MATH_H

Reply:
Good catch - the file was called rbit.h before I decided to just move it to a generic "math.h" header where other functions can be added later.